site/src/components/layout.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Layout
+description: The layout component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - layout
+hero: true
+---
+
+# Layout
+
+The layout component provides a flexible and efficient way to structure the major containers of web pages.
+
+## Try layouts here
+
+This page is utilizing the holy grail layout, but you can use the buttons below to toggle layouts and test them out by resizing the screen. Try the "Toggle container labels" button below to see the css classes on each of the containers inside `.layout`.
+
+<div class="buttons buttons-addons margin-top-sm display-inline-flex">
+  <button class="button" data-set-layout="layout-holy-grail" aria-pressed="true">Holy grail</button>
+  <button class="button" data-set-layout="layout-single">Single</button>
+</div>
+<button class="button margin-top-sm" data-toggle-debug aria-pressed="false">Toggle container labels</button>
+
+## Important concepts
+
+Thankfully, no one needs to be expert to use the `.layout` element. But it does help to understand a few important concepts before beginning.
+
+### Markup structure
+
+The `.layout` class is designed to be place on the `<html>` element itself, while the `layout-body` element is designed to be place on the `<body>` element. All other element are children of `layout-body`. Specific layouts are applied by adding a class with the format `layout-{nameoflayout}` to the `html.layout` element.
+
+Nearly all layouts have the same eight elements, and the `layout-body-hero` element is always optional. While the `layout`, `layout-body`, `layout-body-header`, `layout-body-footer`, and `layout-body-main` elements are always required. The behavior of `layout-body-menu` and `layout-body-aside` depend on the layout.
+
+The basic boilerplate for a layout is as follows:
+
+```html-no-example
+<html class="layout layout-{nameoflayout}">
+	<body class="layout-body">
+		<header style="layout-body-header">Header</header>
+		<section aria-label="hero" class="layout-body-hero">Hero</section>
+		<nav class="layout-body-menu">Menu</nav>
+		<main class="layout-body-main">Main</main>
+		<aside class="layout-body-aside">Aside</aside>
+		<footer class="layout-body-footer">Footer</footer>
+	</body>
+</html>
+```
+
```

Comment:
Grid areas have the potential to make DOM order separate from visual hierarchy, which can cause confusion for keyboard users who need to tab. Can we add a quick advisory note recommending predictable DOM orders?

Reply:
That's a great idea, split Accessibility Concerns into two sections, adding one related to tab order.